src/app.js

```diff
@@ -2,137 +2,243 @@
 /* eslint-disable @typescript-eslint/unbound-method */
 /* eslint-disable @typescript-eslint/no-var-requires */
 // Required imports
-const { ApiPromise, WsProvider } = require('@polkadot/api');
+const { Keyring, ApiPromise, WsProvider } = require('@polkadot/api');
 const PlugRuntimeTypes = require('@plugnet/plug-api-types');
-const Keyring = require('@polkadot/keyring');
-const testingPairs = require('@polkadot/keyring/testingPairs');
+const { Api } = require('@cennznet/api');
 const cli = require('../src/cli.js');
 const selector = require('../src/selector.js');
+const addStaking = require('../src/stakingSetup');
+const { makeTransaction } = require('../src/transaction')
+const { sleep } = require('../src/utils')
 require('console-stamp')(console, 'HH:MM:ss.l');
+let log = require('console-log-level')({ level: 'info' });
+let createStashAccounts = require('../src/stakingSetup')
 
 const APP_SUCCESS = 0;
-const APP_FAIL_TRANSACTION_REJECTED = 1;
 const APP_FAIL_TRANSACTION_TIMEOUT = 2;
 
 async function main (settings) {
+  let result = APP_SUCCESS;
   const config = await setup(settings);
-  const result = await run(config);
-  return result; 
+  while(true) {
```

Comment:
we should make this a configurable flag, like `--forever` versus.
For now it's OK, but eventually the flags should define if we exit on failure or not.